test/functional/bitcoin_cli.py

```diff
@@ -17,10 +17,19 @@ def run_test(self):
         """Main test logic"""
 
         self.log.info("Compare responses from getinfo RPC and `bitcoin-cli getinfo`")
-        cli_get_info = self.nodes[0].cli.getinfo()
+        cli_get_info = self.nodes[0].cli().getinfo()
         rpc_get_info = self.nodes[0].getinfo()
-
         assert_equal(cli_get_info, rpc_get_info)
 
+        self.log.info("Test -stdinrpcpass option")
+        user, password = get_auth_cookie(self.nodes[0].datadir)
+        block_count = self.nodes[0].cli(rpcuser=user, rpcport=rpc_port(0), stdinrpcpass=None).getblockcount(input=password)
```

Comment:
It seems weird to me that `input` is an argument to the RPC, rather than an argument to the bitcoin-cli instance.

It seems possible that an RPC will have `input` as a parameter. Rather less likely that bitcoin-cli will add an argument called `input`.